setup/environment.yml

```diff
@@ -3,77 +3,76 @@ channels:
   - conda-forge
```

Comment:
Lets not change this file unless absolutely necessary

Reply:
Please @737Baron read @CrimsonJacket's comment. Do not change this file unless required. And please never ever remove all version numbers.

Reply:
Undid the env changes